code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -127,6 +118,14 @@
 		src.modules += new /obj/item/weapon/bonesetter(src)
 		src.modules += new /obj/item/weapon/circular_saw(src)
 		src.modules += new /obj/item/weapon/surgicaldrill(src)
+		src.modules += new /obj/item/weapon/gripper/medical(src) //Allows the Medborg to manipulate organs/iv bags/patches/pills and biomass
+		src.modules += new /obj/item/weapon/matter_decompiler(src) //For cleaning and accumulating biomass to deposit at the cloner
+		src.modules += new /obj/item/weapon/soap(src) //For cleaning the disgusting mess that is Medbay
+		src.modules += new /obj/item/weapon/crowbar(src) //For accessing patients in depowered areas

```

Comment:
Mediborgs don't need a crowbar, at all. They can call a human or get the AI to manually turn on an APC.
On the bright side, I hear crowbars are lovely for bashing heads in.